spec/unicorn_wrangler/rss_reader_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../spec_helper'
+require 'unicorn_wrangler/rss_reader'
+
+# Travis is a linux system so we won't test the rss_posix call there,
+# but make no assumptions about running locally.
+SingleCov.covered! uncovered: ENV['TRAVIS'] ? 1 : 6
+
+describe UnicornWrangler::RssReader do
```

Comment:
how about testing the main entrypoint and asserting that works ?
... for linux both should work so we can get to 0 uncovered ?